### PR TITLE
Replace slf4j-log4j12 with slf4j-jdk14

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -288,7 +288,7 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
+      <artifactId>slf4j-reload4j</artifactId>
```
Comment (timja) [inline pom.xml:291]: Would this work instead? not sure if it's being used as a bridge or as an implementation
```suggestion
      <artifactId>slf4j-jdk14</artifactId>
```

Comment (NotMyFault) [reply pom.xml:291]: According to the SLF4J manual, the preferred drop-in replacement is reload4j.

Comment (timja) [reply pom.xml:291]: but do we want this to stay on log4j 1?

Comment (NotMyFault) [reply pom.xml]: Quickly checked locally, jdk14 passes `verify`.

**Review by timja — APPROVED** [review]

**Pull request merged**